src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -320,6 +320,8 @@ object StringUtils {
 
     fun String.isRoman(): Boolean = UtilsPatterns.isRomanPattern.matches(this)
 
+    fun String.isSelectOption(): Boolean = UtilsPatterns.isSelectOptionPattern.matches(this)
```

Comment:
This function should not be a extended function. we only need this in Replace Roman Numerals, we dont want to see this anywhere else when working with strings.